src/app/Scenes/Artwork/Components/ArtworkTombstone.tests.tsx

```diff
@@ -51,12 +60,17 @@ describe("ArtworkTombstone", () => {
     expect(navigate).toHaveBeenCalledWith("/artist/andy-warhol")
   })
 
-  it("redirects to attribution class faq page when attribution class is clicked", () => {
-    const { getByText } = renderWithWrappers(<ArtworkTombstone artwork={artworkTombstoneArtwork} />)
+  it.only("shows the attribution class modal when limited edition set text is tapped", () => {
```

Comment:
is this just accidental?
```suggestion
  it("shows the attribution class modal when limited edition set text is tapped", () => {
```